phoenix/core/settings.py

```diff
@@ -232,6 +232,8 @@
 GITLAB_PRIVATE_TOKEN = os.getenv('GITLAB_PRIVATE_TOKEN')
 GITLAB_POSTMORTEM_DAYS_TO_NOTIFY = list(map(int, os.getenv('GITLAB_POSTMORTEM_DAYS_TO_NOTIFY', '3,7').split(',')))
 
+GOOGLE_ACC = os.getenv('GOOGLE_ACC')
```

Comment:
Where is this env var used? Can it be None without any issue in the code?

Reply:
https://github.com/kiwicom/phoenix/blob/master/phoenix/integration/google.py#L13

yes it won't change the subject in that case. In our setup it is needed.

Reply:
Alright 😄